src/Sylius/Component/Taxation/Model/TaxRateInterface.php

```diff
@@ -42,4 +42,14 @@ public function getCalculator(): ?string;
     public function setCalculator(?string $calculator): void;
 
     public function getLabel(): ?string;
+
+    public function getStartDate(): ?\DateTimeInterface;
+
+    public function setStartDate(?\DateTimeInterface $startDate): void;
+
+    public function getEndDate(): ?\DateTimeInterface;
+
+    public function setEndDate(?\DateTimeInterface $endDate): void;
+
+    public function isInDate(\DateTimeInterface $date): bool;
```

Comment:
I have mixed feelings about this method in general. IMO it's not a part of the model as we have to pass the `$date` argument to it.